internal/proxy/options_test.go

```diff
@@ -95,6 +98,17 @@ func TestProviderURLValidation(t *testing.T) {
 			providerURLString: "https://provider.example.com",
 			expectedSignInURL: "https://provider.example.com/sign_in",
 		},
+		{
+			name:              "redeem string based on providerURL",
+			providerURLString: "https://provider.example.com",
+			expectedRedeemURL: "https://provider.example.com/redeem",
+		},
+		{
+			name:                   "redeem string based on proxyProviderURL",
+			providerURLString:      "https://provider.example.com",
+			proxyProviderURLString: "https://provider-internal.example.com",
```

Comment:
is this field being used anywhere?

Reply:
@shrayolacrayon it's the string that we use to generate the `proxyProviderURL` url 
https://github.com/buzzfeed/sso/blob/dec558663391454c352bf0561a04d604ad5c8372/internal/proxy/options.go#L223-L226 which is used to generate the redeem url: https://github.com/buzzfeed/sso/blob/dec558663391454c352bf0561a04d604ad5c8372/internal/proxy/providers/sso.go#L63